datadog/resource_datadog_monitor.go

```diff
@@ -418,6 +432,32 @@ func resourceDatadogMonitorRead(d *schema.ResourceData, meta interface{}) error
 		d.Set("query_config", queryConfig)
 	}
 
+	// The Datadog API doesn't return old timestamps or support a special value for unmuting scopes
+	// So we provide this functionality by saving values to the state
+	apiSilenced := m.Options.Silenced
+	configSilenced := d.Get("silenced").(map[string]interface{})
+	unmutedScopes := getUnmutedScopes(d)
+
+	// If the scope is in the API response but not in the config
```

Comment:
I think this functionality should be in the update function, not read. The reason for that is that if it's in read, it will create confusing TF plans with the `-1` value that the user never configured anywhere. Does that make sense?

I *think* the above should be possible, but if not, this is still better than what we have currently.